packages/codegen-ui/lib/generate-form-definition/helpers/form-field.ts

```diff
@@ -147,14 +147,17 @@ export function getFormDefinitionInputElement(
   config: StudioGenericFieldConfig,
   baseConfig?: StudioGenericFieldConfig,
 ): FormDefinitionInputElement {
-  const componentType = config.inputType?.type || baseConfig?.inputType?.type;
+  let componentType = config.inputType?.type || baseConfig?.inputType?.type;
 
   if (!componentType) {
-    throw new InvalidInputError('Field config is missing input type');
+    // Gracefully fall back to a TextField if the inputType is no longer available due to a field rename
+    componentType = 'TextField';
```

Comment:
can this be chained to the pre-existing coalesce on line150?

Reply:
I'd rather not chain that far. It gets harder to read.

Reply:
understood, thanks!